OsmAnd/src/net/osmand/plus/settings/preferences/SizePreference.java

```diff
@@ -85,7 +89,8 @@ public CharSequence getSummary() {
 		String persistedString = getValue();
 		if (!persistedString.equals(defaultValue)) {
 			try {
-				persistedString = String.valueOf(Float.parseFloat(persistedString) + 0.01f);
+				final DecimalFormat df = new DecimalFormat("#.####", new DecimalFormatSymbols(Locale.US));
```

Comment:
Strange part.... didn't understand why so complicated

Reply:
Yes, String.valueOf() is redundant